client/src/components/questionsAnswers/D-QuestionsCards.jsx

```diff
@@ -48,18 +55,20 @@ function QuestionsCards(props) {
   }
 
   // TODO: HANDLE PHOTOS IN AXIOS POST REQUEST
-  function handleModalSubmit(text, nickname, userEmail) {
+  function handleModalSubmit(text, nickname, userEmail, photos) {
     const body = {
       body: text,
       name: nickname,
       email: userEmail,
-      photos: ['placeholder'],
+      photos,
     };
     axios.post(`/qa/questions/${question_id}/answers`, body)
+      .then((response) => console.log(response))
```

Comment:
don't forget to remove the console logs later on =)

Reply:
Thanks Kevin! Change is pushed!